bukkit/src/main/java/dev/jsinco/brewery/bukkit/util/MessageUtil.java

```diff
@@ -178,4 +182,10 @@ public static String formatIngredients(Map<Ingredient, Integer> ingredients) {
                 .append(Component.text(severeLevel.toString()).color(NamedTextColor.GOLD))
                 .append(Component.text(remainder.toString()).color(NamedTextColor.BLACK));
     }
+
+    public static @NotNull TagResolver getTimeTagResolver(long timeTicks) {
+        long seconds = (timeTicks % Moment.MINUTE) / Moment.SECOND;
+        long minutes = timeTicks / Moment.MINUTE;
+        return Placeholder.parsed("time", minutes + ":" + seconds);
```

Comment:
I think `0:03` looks nicer instead of `0:3`
```suggestion
        return Placeholder.parsed("time", minutes + ":" + String.format("%02d", seconds));
```
fancier version:
```suggestion
        return Placeholder.parsed("time", String.format("%d:%02d", minutes, seconds));
```